docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -35,21 +29,37 @@ $ ./build_tools/riscv/riscv_bootstrap.sh
 
 ## Configure and build
 
-### Configure on Linux
+### Host configuration
+
+Build and install at least the compiler tools on your host machine, or install them from a binary distribution:
+
+```shell
+$ mkdir build-host ; cd build-host ; mkdir install
+$ cmake -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=./install ..
+$ cmake --build . --target install
+```
+
+Debugging note: if `IREE_LLVMAOT_LINKER_PATH` is set for targeting Android then the build above will fail, and you should run `unset IREE_LLVMAOT_LINKER_PATH`.
+
+### Target configuration
 
 ```shell
+# Back to iree root folder.
 $ mkdir build-riscv ; cd build-riscv
```

Comment:
nit: Similar here. The build-riscv target can be outside of the iree root directory (use -B flag) so the build structure could be similar to other targets.